lib/commands/new.ts

```diff
@@ -19,16 +21,34 @@ export class ${className} implements MigrationInterface {
 `;
 };
 
+export const getMigrationTemplate = (
+  className: string,
+  templateFile?: string
+) => {
```

Comment:
```suggestion
): string => {
```

specify the type returned